packages/node/src/yargs.ts

```diff
@@ -147,6 +148,12 @@ export const yargsOptions = yargs(hideBin(process.argv))
       type: 'boolean',
       default: false,
     },
+    'mmr-db-store': {
```

Comment:
Let's change this to `mmr-store-type` and have it as an enum. We don't know if later on we have the option for another store type.